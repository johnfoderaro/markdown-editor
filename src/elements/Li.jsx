import React from 'react';
import styled from 'styled-components';
import PropTypes from 'prop-types';

const StyledLi = styled.li;

const Li = ({ children }) => (
  <StyledLi>
    {...children}
  </StyledLi>
);

Li.propTypes = { children: PropTypes.node.isRequired };

export default Li;
