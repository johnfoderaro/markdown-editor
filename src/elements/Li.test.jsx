import React from 'react';
import { shallow } from 'enzyme';

import Li from './Li';

let tree;

beforeAll(() => {
  tree = shallow(
    <Li>
      Test
    </Li>,
  );
});

describe('<Li />', () => {
  it('should be defined', () => {
    expect(tree).toBeDefined();
  });
  it('should render correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
