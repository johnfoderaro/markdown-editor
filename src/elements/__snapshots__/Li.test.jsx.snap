// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Li /> should render correctly 1`] = `
<styled.li>
  Test
</styled.li>
`;
